### DISCUSSION TITLE
Where get clangd release notes, about support list for lsp spec?

### DISCUSSION THREAD

**Discussion by Yurii-huang** [clangd/clangd#1350] [Q&A] [answered]
Thank anyone

**Comment by HighCommander4** [comment, marked as answer]
I think the closest thing we have is https://clangd.llvm.org/features (improvements to this page are welcome). There are also clangd sections in the LLVM release notes, e.g. https://releases.llvm.org/15.0.0/tools/clang/tools/extra/docs/ReleaseNotes.html#improvements-to-clangd.

If you're wondering about support for a specific LSP feature, please feel free to ask.

**Reply by Yurii-huang** [reply]
Very useful, thank you
